### PR TITLE
Built data gen functions to create basic sample data for seeding

### PR DESCRIPTION
**Pull request by matgbx**
https://trello.com/c/Yo5vF2rF/3-refactor-data-generator-function-to-take-basic-data-types-s
https://trello.com/c/IHjok5ni/4-refactor-data-gen-function-to-write-to-file-in-chunks-m

added to this PR considering it is still open:

https://trello.com/c/aa3ifxKo/6-create-seed-function-for-postgres-m
https://trello.com/c/zdOIpgOx/7-optimize-seed-function-sm
https://trello.com/c/h9A64FlX/8-add-index-to-reservationsrestaurantid-column-s

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const addReservation = ({
 }) => genReservationSlots({ restaurantId, date })
   .then((slots) => {
     const requestedSlot = slots.reservations.find(item => item.time === time);
-
+    console.log(restaurantId);
```
Comment (nicola24) [inline server/db/index.js:91]: make sure to remove this before deployment

File: server/routers/reservations.js [hunk 2]
```diff
@@ -9,12 +9,15 @@ router.use(bodyParser.json());
 
 router.post('/', (req, res) => {
   // res.set({ 'Access-Control-Allow-Origin': '*' });
+  console.log(req.body);
```
Comment (nicola24) [inline server/routers/reservations.js:12]: same comment as above

**Review by nicola24 — APPROVED** [review]
Good job on seedPG.js and dataGen.js, I left one comment but its just a suggestion

File: server/routers/reservations.js (hunk 2 above)
Comment (matgbx) [reply server/routers/reservations.js:12]: Thanks, there were both for logging the flow to the console. Will remove in next commit.

**Pull request merged**